generators/app/templates/dotnet/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM <%= baseImageName %>
+COPY project.json /app/
+WORKDIR /app
+RUN ["dotnet", "restore"]
```

Comment:
aspnetHelper.js included updating Program.cs to inculde UseUrls. Shouldn't ENV ASPNETCORE_URLS [http://*:80](http://*:80) be added here for that change?